packages/zwave-js/src/lib/controller/ReplaceFailedNodeRequest.ts

```diff
@@ -0,0 +1,138 @@
+import type { Driver } from "../driver/Driver";
+import {
+	FunctionType,
+	MessagePriority,
+	MessageType,
+} from "../message/Constants";
+import {
+	expectedResponse,
+	gotDeserializationOptions,
+	Message,
+	MessageBaseOptions,
+	MessageDeserializationOptions,
+	MessageOptions,
+	messageTypes,
+	priority,
+} from "../message/Message";
+import type { SuccessIndicator } from "../message/SuccessIndicator";
+
+export enum ReplaceFailedNodeStartFlags {
+	OK = 0,
+	/** The replacing process was aborted because the controller  is not the primary one */
+	NotPrimaryController = 1 << 1,
+	/** The replacing process was aborted because no call back function is used */
+	NoCallbackFunction = 1 << 2,
+	/** The replacing process aborted because the node was node found */
+	NodeNotFound = 1 << 3,
+	/** The replacing process is busy */
+	ReplaceProcessBusy = 1 << 4,
+	/** The replacing process could not be started*/
+	ReplaceFailed = 1 << 5,
+}
+
+export enum ReplaceFailedNodeStatus {
+	/* ZW_ReplaceFailedNode callback status definitions */
+	/** The node is working properly (removed from the failed nodes list ). Replace process stopped */
+	NodeOK = 0,
+
+	/* ZW_ReplaceFailedNode callback status definitions */
+	/** The failed node is ready to be replaced
+        and controller is ready to add new node
+        with the nodeID of the failed node.
+        Meaning that the new node must now
+        emit a nodeinformation frame to be
+        included. */
+	FailedNodeReplace = 1,
+	/** The failed node has been replaced. */
+	FailedNodeReplaceDone = 2,
+
+	/** The replace process has failed */
+	FailedNodeReplaceFailed = 3,
+}
```

Comment:
You are right, the enum values are wrong.
```h
/* ZW_ReplaceFailedNode callback status definitions */
#define ZW_FAILED_NODE_REPLACE              3 /* The failed node are ready to be replaced and controller */
                                              /* is ready to add new node with nodeID of the failed node */
#define ZW_FAILED_NODE_REPLACE_DONE         4 /* The failed node has been replaced */
#define ZW_FAILED_NODE_REPLACE_FAILED       5 /* The failed node has not been replaced */
```

Reply:
Fixed